tests/layer/test_watch_your_step.py

```diff
@@ -40,7 +40,9 @@ def test_AttentiveWalk():
 
     output = att_wlk(random_partial_powers).numpy()
 
-    assert np.allclose(output, random_partial_powers.mean(axis=1))
+    np.testing.assert_allclose(
+        output, random_partial_powers.mean(axis=1), rtol=1e-5, atol=1e-8
```

Comment:
does this result in slightly different behaviour with the thresholds?

Reply:
This change here is actually matching the thresholds: annoyingly `allclose` and `assert_allclose` have different defaults for their parameters:

| function | `rtol` | `atol` |
|---|---|---|
| [`allclose`](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html) | 1e-5 | 1e-8 |
| [`assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html)  | 1e-7 | 0 |